### ISSUE TITLE
Add documentation for the different seriestypes

### ISSUE THREAD

**Issue by dlfivefifty** [JuliaPlots/Plots.jl#466]
I couldn't figure out how to use quiver, and it doesn't seem to be in the docs.

**Comment by tbreloff** [comment]
Here's a usage: https://github.com/tbreloff/Plots.jl/issues/262

In short, the `quiver` seriestype expects you to pass in velocity vectors using the `quiver` attribute (with aliases: `add_aliases(:quiver, :velocity, :quiver2d, :gradient)`

You could pass in a 2-tuple `(u,v)` for vectors u/v, or you could pass in a vector of 2-tuples for each point.  Final form:

```julia
quiver(x, y, quiver = (u, v))
```

**Comment by dlfivefifty** [comment]
Would it be possible to add a page to the docs with examples for every `linetype`, and a plot of the output?  Whenever I need something its a bit of a hide-and-go-seek game to figure out the name of the linetype (I don't remember how I figured out `quiver`) and the usage.

I'm more than happy to help by making a template that has the cases I already know.

**Comment by tbreloff** [comment]
Yes this is one of the last things from the documentation meta-issue. I
just haven't gotten to it yet. Any help is appreciated!

**Comment by dpsanders** [comment]
Could there be an alias `vector_field` for `quiver`? 
I find the name `quiver` confusing and un-discoverable, especially for non-native speakers.

**Comment by dpsanders** [comment]
cf #760

**Comment by juliohm** [comment]
Does quiver works with 3D vectors? I am trying to naively call `quiver(x,y,z, quiver=(u,v,w))`, but it doesn't work. Documentation would be helpful.

**Comment by mkborregaard** [comment]
It is not just `quiver` - there is no general documentation for the different seriestypes. I would say opening a seriestypes tab on the plotdocs with pages for each category of seriestypes ("histogram-like", "scatter-like" is a high priority for documentation. All it requires is for someone to take the time to write them.

**Comment by ChasingZenith** [comment]
It is a little bit surprised to me that this issue have not been alleviated since 2016.

I am confused about why `quiver` is an function like plot but only appear in `Series attribute`. I can’t find any clue in the documentation.

An example can tell a thousand words. To make some progress,I suggest that it will be better if
an example illustrating the usage of `quiver` and some useful associated attribute like `aspect_ratio=:equal` in [Example section](http://docs.juliaplots.org/latest/generated/gr/) and so on. 

BTW,
In fact, I searched the documentation for an entire night and I couldn’t find a figure or a description about how to draw a vector.

I search with keyword  “vector”, “vector field”, “arrow” since I did’t even know the word `quiver` beforehand.

Not until I gave up and google it, I thought that `Plots.jl` does not support this feature.
